src/replace.js

```diff
@@ -42,11 +42,12 @@ function replaceElement(element, options) {
   }
 
   const elementClassAttr = element.getAttribute('class') || '';
+  const elementId = element.getAttribute('id');
```

Comment:
I think this should probably be `elementIdAttr` to be consistent with `elementClassAttr`.